modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotRestoreProcess.java

```diff
@@ -86,6 +87,12 @@
     /** Reject operation message. */
     private static final String OP_REJECT_MSG = "Cache group restore operation was rejected. ";
 
+    /** Snapshot restore operation finish message. */
+    private static final String OP_FINISHED_MSG = "Successfully restored cache group(s) from the snapshot [reqId=%s]";
+
+    /** Snapshot restore operation failed message. */
+    private static final String OP_FAILED_MSG = "Failed to restore snapshot cache group [reqId=%s%s]";
```

Comment:
`reqId=%s%s`  why do we mix a request id and message here?

Reply:
What kind of problem do you see here?
This is a common message format for events and logging. Addition %s is to add error message into event message.